src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -610,9 +610,17 @@ def _channel_str(self):
         Generally of the form of "ip:port", like "127.0.0.1:50052".
 
         """
-        if self._channel is not None:
-            return self._channel._channel.target().decode()
-        return ""
+        channel = self._channel
+        while channel is not None:
```

Comment:
Why ``while``?? Why not ``if``??

Reply:
Ok... so this is a way to iterate through nested channel interceptors... this is perfectly fine then. :)